src/collections/members/ankita-sahu/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+name: Ankita Sahu
+position: Contributor
+image_path: ./ankita-sahu.jpeg
+github: SAHU-01
+twitter: asahu_dev
+linkedin: asahudev
+layer5: ff1f8398-e501-4bb2-817f-80edffd0d37b
+location: India
+bio: Ankita Sahu has been a consistent and active contributor to the community. She is passionate about web technologies and has made significant contributions to the Layer5 project.
```

Comment:
@SAHU-01 are you willing to go with this bio?

Reply:
@sudhanshutech no, i'd prefer something like this:

Ankita Sahu is a frontend developer with over 2 years of experience in creating intuitive, user-friendly interfaces using technologies like React, TypeScript, Next.js, and Tailwind CSS. She has worked on a diverse range of projects  and is adept at Agile development practices. Ankita is also a consistent and active contributor to the community, with significant contributions to the Layer5 project.

I'm not sure if you and @siddharthbaleja7 would be ok with this though.

Reply:
make sense. @siddharthbaleja7 will you please update.